### PR TITLE
Added icons

### PR DESCRIPTION
**Pull request by arjund-06**
### Functionality:
Makes the page look a bit more catchy

### Solution:
Used material icons everywhere. Created different components for the Priority and Status icons and added them to the page.

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
ng serve
(or)
Image down below...

image

### CODE REVIEW

**Comment by shivamgoyal7** [comment]
solve conflict asap

**Comment by viv9k** [comment]
Please remove font awesome from everywhere, and also revert the changes that are not part of this feature.

File: worktrolly-ui/package.json [hunk 1]
```diff
@@ -1,57 +1,57 @@
 {
-  "name": "worktrolly-ui",
```
Comment (viv9k) [inline worktrolly-ui/package.json:2]: Please revert this file, as no changes is done here.

File: worktrolly-ui/src/app/app.component.html [hunk 2]
```diff
@@ -2,4 +2,4 @@
       rel="stylesheet">
 <app-header></app-header>
 <app-body></app-body>
-<router-outlet></router-outlet>
+<router-outlet></router-outlet>
```
Comment (viv9k) [inline worktrolly-ui/src/app/app.component.html:5]: Please revert this file as well

File: worktrolly-ui/src/app/body/create-new-session/create-new-session.component.html [hunk 3]
```diff
@@ -110,4 +110,5 @@ <h5 class="active nav-link" style="font-weight:bolder">Create New Task</h5>
         </div>
     </div>
 </div>
-<app-loader *ngIf="enableLoader"></app-loader>
\ No newline at end of file
+<app-loader *ngIf="enableLoader"></app-loader>
+
```
Comment (viv9k) [inline worktrolly-ui/src/app/body/create-new-session/create-new-session.component.html]: no changes done in this file please revert it

File: worktrolly-ui/src/app/body/create-new-sprint/create-new-sprint.component.html [hunk 4]
```diff
@@ -76,8 +76,7 @@ <h5 class="active nav-link" style="font-weight:bolder">Create New Sprint</h5>
                 <div class="row pt-2 pb-5">
                     <div class="col-md-10"></div>
                     <div class="col-md-2">
-                        <button type="submit" class="btn" id="button">Submit</button>
-                    </div>
+                        <button type="submit" class="btn" id="button">Submit</button>                    </div>
```
Comment (viv9k) [inline worktrolly-ui/src/app/body/create-new-sprint/create-new-sprint.component.html:79]: no changes done in this file, please revert it

File: worktrolly-ui/src/app/body/tasks/tasks.component.html [hunk 5]
```diff
@@ -15,6 +15,7 @@ <h5 class="active nav-link" style="font-weight:bolder">Tasks</h5>
                     </ul>
                 </nav>
             </div>
+
```
Comment (viv9k) [inline worktrolly-ui/src/app/body/tasks/tasks.component.html:18]: no changes done in this file please revert it

File: worktrolly-ui/angular.json [hunk 6]
```diff
@@ -1,134 +1,131 @@
 {
```
Comment (viv9k) [inline worktrolly-ui/angular.json:1]: no changes done in this file, please revert it

**Review by viv9k — CHANGES_REQUESTED** [review]
please revert the files where you have not done changes

File: worktrolly-ui/src/app/app.component.html (hunk 2 above)
Comment (arjund-06) [reply worktrolly-ui/src/app/app.component.html:5]: But this file has the google material icon cdn link

File: worktrolly-ui/src/app/body/create-new-sprint/create-new-sprint.component.html (hunk 4 above)
Comment (arjund-06) [reply worktrolly-ui/src/app/body/create-new-sprint/create-new-sprint.component.html:79]: There was error caused due to an incorrect div tag and so @shivamgoyal7 and @sandeep-v1404 had found it was an incorrect div tag and I had to make that change

File: worktrolly-ui/src/app/body/tasks/tasks.component.html (hunk 5 above)
Comment (arjund-06) [reply worktrolly-ui/src/app/body/tasks/tasks.component.html:18]: I actually pulled some code in this branch by mistake after I made a pr and thus some changes which were made by @sandeep-v1404 have also come here.

File: worktrolly-ui/src/app/body/create-new-sprint/create-new-sprint.component.html (hunk 4 above)
Comment (viv9k) [reply worktrolly-ui/src/app/body/create-new-sprint/create-new-sprint.component.html:79]: you can revert these changes, as there is no change

**Review by viv9k — CHANGES_REQUESTED** [review]
Please revert the changes in package-lock.json  and also resolve the comment asap

File: worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.html [hunk 7]
```diff
@@ -2,19 +2,19 @@
...
             <div class="col-md-1 col-2 px-2">
-                {{task.Priority}}
+                <app-priority-icons [Priority]="task.Priority"></app-priority-icons>
             </div>
             <div class="col-md-1 col-2 px-2">
                 {{task.id}}
             </div>
             <div class="col-md-7 col-4 px-2">
-                {{task.Title}}
+                <i class="material-icons">Title</i> {{task.Title}}
```
Comment (viv9k) [inline worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.html:14]: Change i tags to span tag

File: worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.html [hunk 8]
```diff
@@ -2,19 +2,19 @@
...
             </div>
             <div class="col-md-2 col-2 px-2">
-                {{task.Assignee}}
+                <i class="material-icons">perm_identity</i> {{task.Assignee}}
```
Comment (viv9k) [inline worktrolly-ui/src/app/body/tasks/task-cards/task-cards.component.html:17]: Change i tags to span tag

**Review by viv9k — CHANGES_REQUESTED** [review]
please do the required changes here as well

**Review by viv9k — CHANGES_REQUESTED** [review]
Please revert the changes in package-lock.json and angular.json

File: worktrolly-ui/src/app/app.component.html [hunk 9]
```diff
@@ -1,5 +1,3 @@
-<link href="https://fonts.googleapis.com/icon?family=Material+Icons"
-      rel="stylesheet">
```
Comment (viv9k) [inline worktrolly-ui/src/app/app.component.html:2]: Please revert the this file as well

**Review by viv9k — CHANGES_REQUESTED** [review]
please change this as well

File: worktrolly-ui/src/app/app.component.html (hunk 9 above)
Comment (arjund-06) [reply worktrolly-ui/src/app/app.component.html:2]: It's already changed.

**Comment by arjund-06** [comment]
> Please revert the changes in package-lock.json and angular.json

I did that, I have copied older code from the master branch and pasted in this file

**Comment by viv9k** [comment]
> > Please revert the changes in package-lock.json and angular.json
> 
> I did that, I have copied older code from the master branch and pasted in this file

Learn how to revert the file changes

**Comment by viv9k** [comment]
Closed it as this is given in another pull request.

**Pull request closed without merging**